serverless-ssr-reddit/ssr/vue/index.js

```diff
@@ -35,21 +35,10 @@ module.exports = async (req, res) => {
-          
-          </div><div class="feed-item__info"><div class="feed-item__header"><h2 class="feed-item__heading">{{ link.data.title }}</h2><span class="feed_item__short-link">v.redd.it</span></div><div class="feed-item__meta">r/WTF • Posted by <strong>u/Circuit_8</strong></div></div><div class="feed-item__comments"><svg class="section__list-item-icon section__list-item-icon_compact" width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M7.99963 13.9996C7.69365 13.9996 7.39291 13.9811 7.09617 13.9521L2.99986 16V12.4591C1.17245 11.1767 0 9.20932 0 7.00017C0 3.13384 3.58159 0 7.99963 0C12.4177 0 15.9993 3.13409 15.9993 7.00017C15.9993 10.8657 12.4177 13.9996 7.99963 13.9996Z" fill="#999999"></path></svg>1469</div></div></a>
-          
-          
-          </div></main></div>`),
+        data: () => Object.assign({
+          active: !queryParams.fetch
```

Comment:
This is only used to decide if we're on the base route or not. Because of this, I'd suggest the following change:

```suggestion
          isHomePage: !queryParams.fetch
```

We'd then change it in the template too. 🚀